Rias.Interactive/InteractiveService.cs

```diff
@@ -70,20 +68,21 @@ Task MessageReceivedAsync(SocketMessage message)
 
             var messageTask = mtcs.Task;
             var delay = Task.Delay(timeout.Value);
-            var task = await Task.WhenAny(messageTask, delay).ConfigureAwait(false);
+            var task = await Task.WhenAny(messageTask, delay);
 
             _client.MessageReceived -= MessageReceivedAsync;
 
             if (task == messageTask)
-                return await messageTask.ConfigureAwait(false);
+                return messageTask.Result;
```

Comment:
If the was is already awaited, why are you awaiting it again? just return the result.